src/components/TodoItem.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+
+const TodoItem = ({ task, toggleTask, editTask, removeTask }) => {
+  const [edit, setEdit] = useState(false);
+  const [value, setValue] = useState(task.text);
+
+  const editTaskText = () => {
+    editTask(task._id, value);
+    setEdit(false)
+  }
+
+  return (
+    edit
+      ?
+      <div className={task.isCheck ? 'task task_complete' : 'task'} key={task.id}>
```

Comment:
edit form is a separate component
And just the text of the task is another separate component